### PR TITLE
how-to/deploy: add cdc deployment through tiup

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)

add cdc deployment method through tiup.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -1081,6 +1091,80 @@ alertmanager_servers:
   - host: 10.0.1.4
 ```
 
+### 场景 4：通过 TiCDC 同步到下游
+
+#### 部署需求
+
+设置默认部署目录 `/tidb-deploy` 和数据目录 `/tidb-data`，集群启动 TiCDC，可在部署后通过 cdc cli 创建同步任务。
```
Comment (amyangfei) [inline how-to/deploy/orchestrated/tiup.md:1098]:
```suggestion
设置默认部署目录 `/tidb-deploy` 和数据目录 `/tidb-data`，集群启动 TiCDC，可在部署后通过 `cdc cli` 创建同步任务。
```

File: how-to/scale/with-tiup.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ category: how-to
 
 TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
 
-本文介绍如何使用 TiUP 扩容缩容集群中的 TiDB、TiKV、PD 或者 TiFlash 节点。如未安装 TiUP，可参考[升级文档中的步骤](/how-to/upgrade/using-tiup.md#2-在中控机器上安装-tiup)，将集群 import 到 TiUP 环境中，再进行扩容缩容。
+本文介绍如何使用 TiUP 扩容缩容集群中的 TiDB、TiKV、PD、CDC 或者 TiFlash 节点。如未安装 TiUP，可参考[升级文档中的步骤](/how-to/upgrade/using-tiup.md#2-在中控机器上安装-tiup)，将集群 import 到 TiUP 环境中，再进行扩容缩容。
```
Comment (amyangfei) [inline how-to/scale/with-tiup.md:10]:
```suggestion
本文介绍如何使用 TiUP 扩容缩容集群中的 TiDB、TiKV、PD、TiCDC 或者 TiFlash 节点。如未安装 TiUP，可参考[升级文档中的步骤](/how-to/upgrade/using-tiup.md#2-在中控机器上安装-tiup)，将集群 import 到 TiUP 环境中，再进行扩容缩容。
```

File: how-to/scale/with-tiup.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
 
 > **注意：**
 >
-> 添加 TiKV 和 PD 节点和添加 TiDB 节点的步骤类似。
+> 添加 TiKV、PD、CDC 节点和添加 TiDB 节点的步骤类似。
```
Comment (amyangfei) [inline how-to/scale/with-tiup.md:30]:
```suggestion
> 添加 TiKV、PD、TiCDC 节点和添加 TiDB 节点的步骤类似。
```

File: how-to/scale/with-tiup.md [hunk 4]
```diff
@@ -126,7 +126,7 @@ tiup cluster display <cluster-name>
 
 > **注意：**
 >
-> 移除 TiDB 和 PD 节点和移除 TiKV 节点的步骤类似。
+> 移除 TiDB、PD、CDC 节点和移除 TiKV 节点的步骤类似。
```
Comment (amyangfei) [inline how-to/scale/with-tiup.md:129]:
```suggestion
> 移除 TiDB、PD、TiCDC 节点和移除 TiKV 节点的步骤类似。
```

File: reference/tools/ticdc/deploy.md [hunk 5]
```diff
@@ -9,6 +9,107 @@ category: reference
...
 
+本节介绍了在不同场景下如何安装部署 TiCDC，包括以下场景：
+
+- [使用 TiUP 全新部署 TiCDC](#使用-tiup-全新部署-ticdc)
+- [使用 TiUP 在原有 TiDB 集群上新增 TiCDC 组件](#使用-tiup-在原有-tidb-集群上新增-ticdc-组件)
+- [手动在原有 TiDB 集群上新增 TiCDC 组件](#手动在原有-tidb-集群上新增-ticdc-组件)
+
+### 使用 TiUP 全新部署 TiCDC
+
+TiUP Cluster 是适用于 TiDB 4.0 及以上版本的部署工具，目前推荐使用 TiUP Cluster 安装部署 TiCDC v4.0.0-rc.1 及更新版本，部署流程如下：
```
Comment (amyangfei) [inline reference/tools/ticdc/deploy.md:20]:
```suggestion
TiUP Cluster 是适用于 TiDB 4.0 及以上版本的部署工具，部署运行 TiCDC 必须使用 TiDB v4.0.0-rc.1 或更新版本，部署流程如下：
```

File: reference/tools/ticdc/deploy.md [hunk 6]
```diff
@@ -9,6 +9,107 @@ category: reference
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster display test
+    ```
+
+### 使用 TiUP 在原有 TiDB 集群上新增 TiCDC 组件
+
+1. 首先确认当前 TiDB 的版本支持 TiCDC，否则需要先升级 TiDB 集群至 4.0 rc 以上版本。
```
Comment (amyangfei) [inline reference/tools/ticdc/deploy.md:88]:
```suggestion
1. 首先确认当前 TiDB 的版本支持 TiCDC，否则需要先升级 TiDB 集群至 4.0.0 rc.1 或更新版本。
```

**Comment by lichunzhu** [comment]
addressed, PTAL again @amyangfei

File: reference/tools/ticdc/deploy.md [hunk 7]
```diff
@@ -9,6 +9,107 @@ category: reference
...
+
+    ```shell
+    tiup cluster
+    ```
+
+3. 编写 topology 配置文件，保存为 `topology.yaml`。
+
+    可以参考[全量的配置文件模版](https://github.com/pingcap-incubator/tiup-cluster/blob/master/examples/topology.example.yaml)。
+
+    除了部署 TiDB 集群的配置，需要额外在 `cdc_servers` 下配置 cdc servers 所在的 ip（目前只支持 ip，不支持域名）。
```
Comment (yikeke) [inline reference/tools/ticdc/deploy.md:36]:
```suggestion
    除了部署 TiDB 集群的配置，需要额外在 `cdc_servers` 下配置 CDC 服务器所在的 ip（目前只支持 ip，不支持域名）。
```

File: how-to/deploy/orchestrated/tiup.md [hunk 8]
```diff
@@ -525,6 +525,16 @@ tiflash_servers:
...
+# cdc_servers:
+#   - host: 10.0.1.20
+#     ssh_port: 22
+#     port: 8300
+#     deploy_dir: "/tidb-deploy/cdc-8300"
+#     log_dir: "/tidb-deploy/cdc-8300/log"
+#     numa_node: "0,1"
+#   - host: 10.0.1.21
+#   - host: 10.0.1.22
+
```
[the remark this answers is not in the document]
Comment (yikeke) [reply how-to/deploy/orchestrated/tiup.md:538]: Ok.

**Pull request merged**